install-labkey.bash

```diff
@@ -695,7 +695,13 @@ function step_postgres_configure() {
     sudo DEBIAN_PRIORITY=critical DEBIAN_FRONTEND=noninteractive apt-get update
     # Postgresql 12 included in Ubuntu 20.04
     if [ "$POSTGRES_SVR_LOCAL" == "TRUE" ]; then
-      sudo DEBIAN_PRIORITY=critical DEBIAN_FRONTEND=noninteractive apt-get -y install postgresql-12
+      if [ "$(platform_version)" == "20.04" ]; then
+        sudo DEBIAN_PRIORITY=critical DEBIAN_FRONTEND=noninteractive apt-get -y install postgresql-12
+      fi
+      # Postgresql 14 included in Ubuntu 22.04
```

Comment:
if its included, why are we installing it...?

Reply:
What is meant by this comment is that Postgresql 14 is the only version available in the default Ubuntu APT repos